backend/main.go

```diff
@@ -25,7 +25,13 @@ func main() {
 		log.Fatalln("Could not open Database", err)
 	}
 
-	var store = sessions.NewCookieStore([]byte(SECRET_KEY))
+	secretKey, ok := os.LookupEnv("SECRET_KEY")
```

Comment:
Can we rename `SECRET_KEY` to  something like `COOKIE_SECRET`? `SECRET_KEY` can be easily mistaken to github's secret key env